storybook/src/components/Tabs/Tabs.stories.tsx

```diff
@@ -22,6 +24,15 @@ const SlowPanel = ({ children }: PropsWithChildren) => {
 const meta = {
   title: 'Components/Containers/Tabs',
   component: Tabs,
+  argTypes: {
+    activeTab: {
+      control: {
+        type: 'number',
+        min: 0,
+        max: 3,
```

Comment:
This value can be made dynamic if we store the example content in an object, see other comment.